bakery/src/tasks/checksum-book.js

```diff
@@ -0,0 +1,39 @@
+const dedent = require('dedent')
+
+const task = () => {
+  return {
+    task: 'checksum book',
+    config: {
+      platform: 'linux',
+      image_resource: {
+        type: 'docker-image',
+        source: {
+          repository: 'openstax/cops-bakery-scripts',
+          tag: 'dev-marvin'
+        }
+      },
+      inputs: [
+        { name: 'book' },
+        { name: 'baked-book' }
+      ],
+      outputs: [{ name: 'checksum-book' }],
+      run: {
+        path: '/bin/bash',
+        args: [
+          '-cxe',
+          dedent`
+          exec 2> >(tee checksum-book/stderr >&2)
+          collection_id="$(cat book/collection_id)"
+          cp -r baked-book/* checksum-book
+          book_dir="checksum-book/$collection_id"
+          mkdir "$book_dir/baked"
+          find "baked-book/$collection_id/" -maxdepth 1 -type f -exec cp {} $book_dir/baked \;
+          python /code/scripts/checksum-resources.py "$book_dir"
```

Comment:
This may tie into some work that Thomas is doing for cnx #960 around cleaning up some of our `cp -r input` operations for efficiency, but would it be possible to leave the inputs in `baked-book` and just build the resource outputs in `checksum-book` (the python script would need to take an additional argument I guess)?

I may be putting the cart before the horse here, though, so maybe worth briefly syncing with Thomas to see how perhaps (?) a minor adjustment here could align with the task changes he'll be making.